code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -343,6 +343,57 @@
 		list(mode_name="dual coil",	burst=2, move_delay=5, accuracy = list(-2,-3), dispersion = list(20))
 		)
 
+/obj/item/weapon/gun/projectile/automatic/g11
+	name = "G11"

```

Comment:
Convention is that guns should have a generic name. Specific name is put in the description.